numpy/core/fromnumeric.py

```diff
@@ -3237,12 +3237,14 @@ def around(a, decimals=0, out=None):
     return _wrapfunc(a, 'round', decimals=decimals, out=out)
 
 
-def _mean_dispatcher(a, axis=None, dtype=None, out=None, keepdims=None):
+def _mean_dispatcher(a, axis=None, dtype=None, out=None, keepdims=None,
+                     where=None):
     return (a, out)
 
 
 @array_function_dispatch(_mean_dispatcher)
-def mean(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
+def mean(a, axis=None, dtype=None, out=None, keepdims=np._NoValue,
+         where=np._NoValue):
```

Comment:
Let's make this keyword-only?
```suggestion
         *, where=np._NoValue):
```

Reply:
Thanks, good suggestion! I also made the `where` argument keyword-only in `var(...)`.